packages/amplify-graphql-transformer-core/src/migration/migrators/generators/index.ts

```diff
@@ -0,0 +1,45 @@
+import { parseValue } from "graphql"
+import { defaultProviderMap } from "../auth"
+
```

Comment:
A number of the functions in this file already exist in graphql-transformer-common. For example, search the codebase fo `makeDirective`.